_data/roadmap.yml

```diff
@@ -1,60 +1,60 @@
 - year: 2014
   accomplishments:
     - name: Launched on Bitcointalk
-      date: April 18, 2014
+      date: 18 April 2014
```

Comment:
You're right. My bad. I was on mobile and thought that I had seen the dates formatted that way previously.